frontend/pages/hosts/details/cards/HostSoftwareLibrary/HostSoftwareLibrary.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useCallback, useContext, useMemo, useState } from "react";
```

Comment:
TODO: Rename file directory. fromHostInstallersTable to HostSoftwareLibraryTable